contrib/scripts/ha/README

```diff
@@ -16,7 +16,7 @@ Before you run ha_make_slavedb.sh, you should configure some parameters in ha_ma
   - backup_option    : Additional options for cubrid backupdb. Optional.
   - restore_option   : Additional options for cubrid restoredb. Optional.
   - scp_option       : scp utility options. ha_make_slavedb.sh uses 'scp' to copy a backup volume from a master to a slave. The default values of '-l 16M' means there's a network limitation of 16MB.
-
+  - ssh_port         : port number for scp and ssh utility.
```

Comment:
All the other options are classified into three categories in `example)` below, which is mandatory, conditional and optional. How about adding `ssh_port`?

Reply:
yes, it is better to add ssh_port to below example. I will fix